frontend/packages/ux-editor/src/components/Properties/PageConfigPanel/PageConfigPanel.tsx

```diff
@@ -28,14 +33,28 @@ export const PageConfigPanel = () => {
     layoutNameTextResourceSelector,
   );
   const layoutNameText = layoutNameTextResource?.value;
-
   const headingTitle = !layoutIsSelected
     ? t('right_menu.content_empty')
     : layoutNameText ?? selectedFormLayoutName;
-
   const layout = useFormLayouts()[selectedFormLayoutName];
   const hasDuplicatedIds = duplicatedIdsExistsInLayout(layout);
 
+  const layouts: Record<string, IInternalLayout> = useFormLayouts();
+
+  const layoutPages: FormLayoutPage[] = Object.keys(layouts).map((key) => ({
+    page: key,
+    data: layouts[key],
+  }));
```

Comment:
Can this be moved inside `findLayoutsContainingDuplicateComponents`? 😊
I see that this code is duplicated here and in `DesignView.tsx` 
```suggestion
```

Reply:
fixed